<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="NZ5CG95013MY" failures="1" tests="4" name="Test" time="5.458" errors="0" timestamp="2021-10-31T22:35:45 NZDT">
  <testcase classname="getRequest.TC02a_searchValidateCharityPosTest" name="testGetCharity1" time="4.138"/>
  <testcase classname="getRequest.TC02b_searchValidateCharityNegTest" name="testGetCharity2" time="0.238"/>
  <testcase classname="getRequest.TC03a_ValidateQueryBookPass" name="testGetBookValidate1" time="0.595"/>
  <testcase classname="getRequest.TC03b_ValidateQueryBookFail" name="testGetBookValidate2" time="0.408">
    <failure type="java.lang.AssertionError" message="1 expectation failed.
JSON path Subcategories[7].Subcategories[9].Subcategories[3].Name doesn&amp;apos;t match.
Expected: Disorders &amp;amp; Diseases
  Actual: Disorders &amp;amp; diseases
">
      <![CDATA[java.lang.AssertionError: 1 expectation failed.
JSON path Subcategories[7].Subcategories[9].Subcategories[3].Name doesn't match.
Expected: Disorders & Diseases
  Actual: Disorders & diseases

at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:105)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:59)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:263)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:493)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:674)
at org.codehaus.groovy.runtime.callsite.PlainObjectMetaMethodSite.doInvoke(PlainObjectMetaMethodSite.java:43)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite$PogoCachedMethodSiteNoUnwrapNoCoerce.invoke(PogoMetaMethodSite.java:193)
at org.codehaus.groovy.runtime.callsite.PogoMetaMethodSite.callCurrent(PogoMetaMethodSite.java:61)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callCurrent(AbstractCallSite.java:185)
at io.restassured.internal.ResponseSpecificationImpl.body(ResponseSpecificationImpl.groovy:261)
at io.restassured.specification.ResponseSpecification$body$1.callCurrent(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.body(ResponseSpecificationImpl.groovy:108)
at io.restassured.internal.ValidatableResponseOptionsImpl.body(ValidatableResponseOptionsImpl.java:244)
at io.restassured.internal.ValidatableResponseImpl.super$2$body(ValidatableResponseImpl.groovy)
at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)
at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)
at org.codehaus.groovy.runtime.ScriptBytecodeAdapter.invokeMethodOnSuperN(ScriptBytecodeAdapter.java:144)
at io.restassured.internal.ValidatableResponseImpl.body(ValidatableResponseImpl.groovy:298)
at io.restassured.internal.ValidatableResponseImpl.body(ValidatableResponseImpl.groovy)
at getRequest.TC03b_ValidateQueryBookFail.testGetBookValidate2(TC03b_ValidateQueryBookFail.java:42)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 42 stack frames]]>
    </failure>
  </testcase> <!-- testGetBookValidate2 -->
</testsuite> <!-- Test -->
